EDU 189: Add return for Send Query value (#2138)
temporalio/documentation: docs-src/go/how-to-send-a-query-to-a-workflow-execution-in-go.md

// ...
```

> The `response` returned by the Query needs to be decoded into a `result`.

Reviewer: ```suggestion
The value of `response` returned by the Query needs to be decoded into `result`.
```
Per https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements.

> The `response` returned by the Query needs to be decoded into a `result` string.
> Since this is a future, use `Get()` on the `response` to get a readable string.

Reviewer 2: ```suggestion
The `response` returned by the Query needs to be decoded into a `result`.
Since this is a future, use `Get()` on the `response` to get a readable value such as a string in this example.
```

Result doesn't have to be a string

> Since this is a future, use `Get()` on the `response` to get the result such as a string in this example.

Reviewer: ```suggestion
Because this is a future, use `Get()` on `response` to get the result, such as a string in this example.
```
Per https://developers.google.com/style/word-list#since.
